specs/gradle/gradle.spec

```diff
@@ -61,9 +61,9 @@ Source100:            checksum.sha512
 
 BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:        jdk11 git
+BuildRequires:        jdk13 git
```

Comment:
We don't longer provide non-LTS packages with Java. Currently, the repository contains only JDK8 and JDK11 packages.